package.json

```diff
@@ -26,7 +26,7 @@
     "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore \"**/*.{cjs,js,md,mjs}\"",
     "lint:prettier": "prettier --cache --cache-location .cache/prettier --cache-strategy content --check \"**/*.{cjs,js,json,md,mjs,scss,yaml,yml}\"",
     "lint:scss": "stylelint \"**/*.{md,scss}\"",
-    "check-links": "hyperlink --canonicalroot https://design-system.service.gov.uk/ --internal --root deploy/public --skip \"govuk/all.css\" --skip \"govuk-frontend/all.css\" --skip \"govuk-frontend/all.js\" --skip \"<YOUR-DOMAIN>\" $(glob \"deploy/public/**/*.html\") | tap-spot"
+    "check-links": "hyperlink --canonicalroot https://design-system.service.gov.uk/ --internal --root deploy/public --skip \"govuk/all.css\" --skip \"govuk-frontend/all.css\" --skip \"govuk-frontend/all.js\" --skip \"<YOUR-DOMAIN>\" $(glob \"deploy/public/**/*.html\")"
```

Comment:
Question: without the report does a fail register as such? I'm running locally and the output is pretty annoying - you just get the summary at the end with how many fails there were, then need to scan up 2000+ entries to find the `not ok`.

Reply:
Yeah if you run `echo $?` after it's finished you'll see exit code `1` not `0` when it errors

Output is incredibly annoying but `tap-spot` was flaky too and other reporters don't look maintained 😣
https://github.com/sindresorhus/awesome-tap#reporters